modules/sdk-core/src/bitgo/staking/stakingWallet.ts

```diff
@@ -458,18 +459,6 @@ export class StakingWallet implements IStakingWallet {
       }
     }
 
-    if (buildParams?.aptosCustomTransactionParams) {
```

Comment:
Was this deleted intentionally? This is related to Aptos, but your ticket is related to Avax